### ISSUE TITLE
allow host admin to choose which expense payout options are visible

### ISSUE THREAD

**Issue by emberwry** [opencollective/opencollective#5566]
for the strategic priority of simplification-

user is often a first-time user of OC submitting their first expense. they would prefer to receive their money, for instance, via cashapp, but don't see that as an option. so! they go to "other payment method" in the drop down, put in their payout method which we cannot accomodate, and think they're done. they are not! and now we have to go back and forth to get them to edit and update the expense, when it's someone who's not already familiar with the platform. noting also that this hits less tech-comfortable and underbanked communities hardest.

proposed solution: remove the option to do that! funnel them exclusively into the viable options. a question is when do we use that option (i know @alinamanko does sometimes for reconciliations)

**Comment by alinamanko** [comment]
I agree that we need to get this issue addressed as it causes a lot of unnecessary back and forth with the submitter of the expense. However, I wonder if there is a way to keep the Other option open but slightly altered? It is still used for things like transfers between projects, payments from one collective to another and in instances where expense goes directly through the bank and so just needs to be marked as paid rather than the money physically moving from the host bank account. 

A proposed alternative solution - when someone selects another there will be a form with multi-selection of reasons for transfer and/or notification saying we **do not** support venmo or cash apps transfer. However, that particular one is very host specific for OCF. 

I would love to hear from others on better ways to provide cleaner options to solve this issue.

**Comment by emberwry** [comment]
can each of those just be their own options instead of an 'other' and fill in the blank scenario? like click the drop down, first options are paypal and wise (or what have you) and then you also see options like "collective balance" and "reconciliation" ?

**Comment by alinamanko** [comment]
Maybe we could get a better job in defining those options, and as these haven't been of frequent use up until relatively recently it may be something to consider. However, we do have these exceptions I mentioned earlier so I would love to see if there is a way for us to capture it better and provide the right clarity to limit those random selections of Other by users.

**Comment by Betree** [comment]
We already have the ability to disable the "Other" payout method on a per-host basis (`settings.disableCustomPayoutMethod`).

Reading at the above proposals, I think we can summarize the *best solution for this problem* as: **Provide a way for host admins to define manual payout methods.**

As an MVP, we could also display a small text with instructions in the expense flow when users pick "Other", which would be defined by host admins.

**Comment by alinamanko** [comment]
Thank you for this feedback @Betree. I think if we can have a message that can be customised per host needs it would be a really great option to start with and for us to test whether it can work out. As per my previous feedback, I would be reluctant to disable the Other display.
